frontend/src/__tests__/cypress/cypress/e2e/projects/ProjectList.cy.ts

```diff
@@ -108,6 +137,137 @@ describe('Data science projects details', () => {
+      projectListPage.findSortButton('Name').should(be.sortAscending);
+
+      projectListPage.findSortButton('Created').click();
+      projectListPage.findSortButton('Created').should(be.sortAscending);
+      projectListPage.findSortButton('Created').click();
+      projectListPage.findSortButton('Created').should(be.sortDescending);
+    });
+  });
+
+  it('Validate that clicking on switch toggle will open modal to stop workbench', () => {
+    cy.interceptK8sList(ProjectModel, mockK8sResourceList([mockProjectK8sResource({})]));
+    cy.interceptK8s('PATCH', NotebookModel, mockNotebookK8sResource({})).as('stopWorkbench');
+    cy.interceptK8sList(PodModel, mockK8sResourceList([mockPodK8sResource({})]));
+    cy.interceptK8s(RouteModel, mockRouteK8sResource({ notebookName: 'test-notebook' })).as(
+      'getWorkbench',
+    );
+    cy.interceptK8sList(
+      { model: NotebookModel, ns: 'test-project' },
+      mockK8sResourceList([
+        mockNotebookK8sResource({
+          opts: {
+            spec: {
+              template: {
+                spec: {
+                  containers: [
+                    {
+                      name: 'test-notebook',
+                      image: 'test-image:latest',
+                    },
+                  ],
+                },
+              },
+            },
+            metadata: {
+              name: 'test-notebook',
+              labels: {
+                'opendatahub.io/notebook-image': 'true',
+              },
+              annotations: {
+                'opendatahub.io/image-display-name': 'Test image',
+              },
+            },
+          },
+        }),
+      ]),
+    );
+    projectListPage.visit();
+    cy.wait('@getWorkbench');
+    const projectTableRow = projectDetails.getProjectTableRow('Test Project');
+    projectTableRow.findEnableSwitch().click();
+
+    //stop workbench
+    notebookConfirmModal.findStopWorkbenchButton().should('be.enabled');
+    cy.interceptK8s(
+      NotebookModel,
+      mockNotebookK8sResource({
+        opts: {
+          metadata: {
+            labels: {
+              'opendatahub.io/notebook-image': 'true',
+            },
+            annotations: {
+              'kubeflow-resource-stopped': '2023-02-14T21:45:14Z',
+              'opendatahub.io/image-display-name': 'Test image',
+            },
+          },
+        },
+      }),
+    );
+    cy.interceptK8sList(PodModel, mockK8sResourceList([mockPodK8sResource({ isRunning: false })]));
+
+    notebookConfirmModal.findStopWorkbenchButton().click();
+    cy.wait('@stopWorkbench').then((interception) => {
+      expect(interception.request.body).to.containSubset([
+        {
+          op: 'add',
+          path: '/metadata/annotations/kubeflow-resource-stopped',
+        },
+      ]);
+    });
+    projectTableRow.findHaveNotebookStatusText().should('have.text', 'Stopped ');
+    projectTableRow.findNotebookRouteLink().should('have.attr', 'aria-disabled', 'true');
```

Comment:
Does this work instead of explicitly checking the `aria-disabled` attribute:
```suggestion
    projectTableRow.findNotebookRouteLink().should('be.disabled');
```

Reply:
Doing this causes an AssertionError